config.cfg

```diff
@@ -0,0 +1,52 @@
+# Copyright 2016-2016 the openage authors. See copying.md for legal info.
+
+set TOGGLE_CONSOLE `
```

Comment:
I'm not sure about where to put the file, I think the best would be either in the same place as the executable/launcher or in a subdirectory for config file. 

As for the extension, I used `.cfg` since it is a config file but anything could do. I prefer `.oac` over `.oacvar` since it is shorter.